packer/HWIMO-BUILD

```diff
@@ -4,7 +4,13 @@
 # input (enviromental variable):
 # * OS_VER        -   ubuntu-14.04 or ubuntu-16.04
 # * BUILD_TYPE    -   vmware or virtualbox
```

Comment:
maybe add some info there stating the default value is virtualbox, as the code indicts.